packages/slider/src/Slider.module.scss

```diff
@@ -2,10 +2,9 @@
     --slider-scrollbar-color: #b8b8b8;
     --slider-scrollbar-border-width: 0.35rem;
     --slider-scrollbar-border-radius: 2rem;
-    --slider-track-justify-content: flex-start;
-    --slider-slide-spacing: 1rem;
-    --slider-scroll-snap-type: x mandatory;
-    --slider-scroll-snap-align: center;
+    --slider-slide-spacing: 0;
```

Comment:
Maybe `--slider-gap` would be a better name?

Reply:
Much better